### PR TITLE
Added Python examples. 

### PR DESCRIPTION
**Pull request by MishaJexia**

### CODE REVIEW

File: docs/auth/README.md [hunk 1]
```diff
@@ -25,7 +25,33 @@ For example, let's show how to provide **Read Only** access to our `orders` Data
...
+      secret=JEXIA_API_SECRET,
+  )
+  res = client.request(
+          method='GET', 
+          url='/ds/orders',
+          cond='[{"field":"dislike"},"=",true]',
+          outputs='["id","total","title"]'
+        ) 
+  print(res)
+
```
Comment (velp) [inline docs/auth/README.md:51]: will be better to use pep8 formating for code examples:
1. use 4 spaces everywhere
2. use empty line between modules and if section
3. use correct number of spaces

So this example should looks like:

```python
from jexia_sdk.http import HTTPClient

JEXIA_PROJECT_ID = ''
JEXIA_API_KEY = ''
JEXIA_API_SECRET = ''

if __name__ == '__main__':
    client = HTTPClient()
    client.auth_consumption(
        project=JEXIA_PROJECT_ID,
        method='apk',
        key=JEXIA_API_KEY,
        secret=JEXIA_API_SECRET,
    )
    res = client.request(
        method='GET',
        url='/ds/orders',
        cond='[{"field":"dislike"},"=",true]',
        outputs='["id","total","title"]'
    )
    print(res)
```

File: docs/auth/README.md [hunk 2]
```diff
@@ -94,7 +120,36 @@ After this, you need go to the **Policies** section and create a new policy, ens
...
+  )
+  # Make future requests with user access rights
+  res = client.request(
+          method='GET', 
+          url='/ds/orders',
+          cond='[{"field":"dislike"},"=",true]',
+          outputs='["id","total","title"]'
+        ) 
+  print(res)
+
```
Comment (velp) [inline docs/auth/README.md:149]: the same here - pep8

File: docs/auth/README.md [hunk 3]
```diff
@@ -109,17 +164,20 @@ jexiaClient().init({
...
+);  
 ```
 </template>
 <template v-slot:bash>
 
 ``` bash
 # Environment variables to be set
 export PROJECT_ID=<project_id>
-export TEST_USER=<user_here>
+export TEST_USER=admin@x.com
```
Comment (velp) [inline docs/auth/README.md:180]: Why did you change this? All other variables are template why do we need to have fake email address here?

File: docs/automation/README.md [hunk 4]
```diff
@@ -23,15 +23,37 @@ We expect your application will be managing endpoints to generate and accept a n
...
+          url='/ums/resetpassword/'
+        ) 
+  #to apply changes
+  res = client.request(
+          method='POST',
+          data={"new_password": "jexia_super"},
+          url='ums/resetpassword/token' #token - user will get by email if you have Integration for SMTP
+        ) 
+  print(res)
+
```
Comment (velp) [inline docs/automation/README.md:44]: pep8 again, and line 41 longer than 80 symbols (all IDEs will mark this line is incorrect)

File: docs/automation/README.md [hunk 5]
```diff
@@ -46,10 +68,11 @@ curl
 
 
 # To apply new password
+# token - user will get by email if you have Integration for SMTP
 curl 
 -X POST -d '{
   "new_password": "jexia_super"
-}' "https://$PROJECT_ID.app.jexia.com/ums/resetpassword/token" | jq .
+}' "https://$PROJECT_ID.app.jexia.com/ums/resetpassword/"+token | jq .
```
Comment (velp) [inline docs/automation/README.md:75]: Are you sure that `+token` will work? Bash it's not programming language and you cannot just add the string like this

File: docs/fileset/README.md [hunk 6]
```diff
@@ -38,7 +38,40 @@ Each request consists of two parts:
...
+      password=USER_PASSWORD
+  )
+  data = {
+     "email":"user@email"
+  }
+  files = open('./file.txt','rb')
+  
+  res = client.request(
+          method='POST',
+          data=user,
```
Comment (velp) [inline docs/fileset/README.md:66]: is will not work. In this line you use variable `user`, but above I see only variable `data`

File: docs/fileset/README.md [hunk 7]
```diff
@@ -38,7 +38,40 @@ Each request consists of two parts:
...
+  client.auth_consumption(
+      project=JEXIA_PROJECT_ID,
+      method='ums',
+      email=USER_EMAIL,
+      password=USER_PASSWORD
+  )
+  data = {
+     "email":"user@email"
+  }
+  files = open('./file.txt','rb')
```
Comment (velp) [inline docs/fileset/README.md:62]: Did you test it locally? I think it will not work, file should be read with context manager, like:
```python
...
with open('./file.txt','rb') as file:
    res = client.request(
        method='POST',
        data=user,
        files=file,
        url='/fs/<fileset-name>'
    )
    print(res)
```

File: docs/fileset/README.md [hunk 8]
```diff
@@ -160,22 +212,39 @@ Updating a fileset works the same way as a dataset.
...
  .update({ "isDefaultImage": false })  
  .where(field => field("name").isEqualTo("companyLogo.png"))
- .subscribe();  
+ .subscribe(data=>{}, err=>{});  
 ```
+</template>
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/fileset/README.md:228]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/fileset/README.md [hunk 9]
```diff
@@ -128,15 +163,32 @@ If you do not have real-time available for your project, you can do a request to
 It's possible to query a fileset in the same way as datasets. The only difference is when performing an insert query, you need to use the `upload()` method instead.
 All fileset records will have fields such as name, size, URL, etc. These fields, as well as any custom fields, can be used for select queries. Please keep in mind that these default fields are immutable.
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/fileset/README.md:170]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/automation/README.md [hunk 10]
```diff
@@ -23,15 +23,37 @@ We expect your application will be managing endpoints to generate and accept a n
 
 An example of this is below:
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/automation/README.md:30]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/fileset/README.md [hunk 11]
```diff
@@ -187,15 +256,28 @@ $ curl -s
...
+</template>
+<template v-slot:py>
+
+``` py
+  res = client.request(
+          method='DELETE',
+          url='/fs/<fileset-name>',
+          cond=[{"field":"size"},">",1024000]
+        ) 
+  print(res)
```
Comment (velp) [inline docs/fileset/README.md:277]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/get-started/README.md [hunk 12]
```diff
@@ -91,16 +91,52 @@ Now let's make simple CRUD to access our Dataset. For this, you can use the REST
...
+</template>
+<template v-slot:bash>
+
+``` bash
+# Environment variables to be set
+export PROJECT_ID=<project_id>
+export API_KEY=<key_here>
+export API_SECRET=<secret_here>
+export TEST_USER=<user_here>
+export TEST_USER_PSW=<password_here>
```
Comment (velp) [inline docs/get-started/README.md:129]: Why do we need these variables on this step? As I see they are part of CURL example (lines 230-256 in this file)

File: docs/get-started/README.md [hunk 13]
```diff
@@ -220,6 +256,32 @@ curl -H "Authorization: Bearer $UMS_TOKEN"
...
+      key=JEXIA_API_KEY,
+      secret=JEXIA_API_SECRET,
+  )
+  res = client.request(
+          method='GET', 
+          url='/ds/orders',
+          cond='[{"field":"dislike"},"=",true]',
+          outputs='["id","total","title"]'
+        ) 
+  print(res)
```
Comment (velp) [inline docs/get-started/README.md:281]: pep8 again

File: docs/ums/README.md [hunk 14]
```diff
@@ -21,15 +21,49 @@ jexiaClient().init({
...
+        "city": "Apeldoorn",
+        "country": "NL"
+    }
+  }
+  #to request password reset
+  res = client.request(
+          method='POST',
+          data=user,
+          url='/ums/signup'
+        )
```
Comment (velp) [inline docs/ums/README.md:64]: pep8

File: docs/ums/README.md [hunk 15]
```diff
@@ -125,15 +183,28 @@ export UMS_TOKEN=`curl -X POST -d '{
...
+ums.getUser('Elon Musk').subscribe(user => {}, error=>{});    
 // via email 
-const user = await ums.getUser('elon@tesla.com'); 
+ums.getUser('elon@tesla.com').subscribe(user => {}, error=>{});   
 ```
+</template>
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/ums/README.md:199]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/ums/README.md [hunk 16]
```diff
@@ -153,12 +224,26 @@ You can do user management via CRUD operations. This method is mainly for the cu
...
 ``` js
-ums.deleteUser('Elon@tesla.com', password); 
+ums.deleteUser('Elon@tesla.com', password)
+.subscribe(user => {}, error=>{});    
+```
+</template>
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/ums/README.md:238]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/ums/README.md [hunk 17]
```diff
@@ -176,12 +261,32 @@ There are two ways to change the password for a user by using their old password
...
-ums.changePassword('Elon@tesla.com', oldPassword, newPassword); 
+ums
+.changePassword('Elon@tesla.com', oldPassword, newPassword)
+.subscribe(user => {}, error=>{});   
 ```
+</template>
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/ums/README.md:276]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/ums/README.md [hunk 18]
```diff
@@ -153,12 +224,26 @@ You can do user management via CRUD operations. This method is mainly for the cu
...
+  res = client.request(
+          method='DELETE',
+          url='/ums/user/'
+        ) 
+  print(res)
```
Comment (velp) [inline docs/ums/README.md:243]: will return nothing here, you can remove this line

File: docs/ums/README.md [hunk 19]
```diff
@@ -200,15 +305,43 @@ curl
...
+ums
+.resetPassword(Token, newPassword)
+.subscribe(user => {}, error=>{});   
+```
+
+</template>
+<template v-slot:py>
+
+``` py
+...
```
Comment (velp) [inline docs/ums/README.md:327]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/ums/README.md [hunk 20]
```diff
@@ -200,15 +305,43 @@ curl
...
+  res = client.request(
+          method='POST',
+          data=user,
+          url='/ums/resetpassword/'
+        ) 
+  #to apply changes
+  res = client.request(
+          method='POST',
+          data={"new_password": "jexia_super"},
+          url='ums/resetpassword/token' #token - user will get by email if you have Integration for SMTP
```
Comment (velp) [inline docs/ums/README.md:341]: pep8 - long line

File: docs/ums/README.md [hunk 21]
```diff
@@ -240,23 +373,36 @@ For this you need to create a policy with the following values:
...
+</template>
+<template v-slot:py>
+
+``` py
+  res = client.request(
+          method='GET',
+          url='/ums/users',
+          cond='[....]'
+        ) 
+  print(res)
```
Comment (velp) [inline docs/ums/README.md:402]: I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

File: docs/auth/README.md (hunk 1 above)
Comment (MishaJexia) [reply docs/auth/README.md:51]: fixed.

File: docs/auth/README.md (hunk 2 above)
Comment (MishaJexia) [reply docs/auth/README.md:149]: fixed

File: docs/auth/README.md (hunk 3 above)
Comment (MishaJexia) [reply docs/auth/README.md:180]: It was a minute of creativity...

File: docs/automation/README.md (hunk 4 above)
Comment (MishaJexia) [reply docs/automation/README.md:44]: fixed

File: docs/automation/README.md (hunk 5 above)
Comment (MishaJexia) [reply docs/automation/README.md:75]: put it back

File: docs/fileset/README.md (hunk 6 above)
Comment (MishaJexia) [reply docs/fileset/README.md:66]: fixed.

File: docs/fileset/README.md (hunk 7 above)
Comment (MishaJexia) [reply docs/fileset/README.md:62]: changed.

File: docs/fileset/README.md (hunk 8 above)
Comment (MishaJexia) [reply docs/fileset/README.md:228]: I will work on this in the next update.

File: docs/fileset/README.md (hunk 9 above)
Comment (MishaJexia) [reply docs/fileset/README.md:170]: same here

File: docs/automation/README.md (hunk 10 above)
Comment (MishaJexia) [reply docs/automation/README.md:30]: will add in next release

File: docs/fileset/README.md (hunk 11 above)
Comment (MishaJexia) [reply docs/fileset/README.md:277]: will add next release

File: docs/get-started/README.md (hunk 12 above)
Comment (MishaJexia) [reply docs/get-started/README.md:129]: changed.

File: docs/get-started/README.md (hunk 13 above)
Comment (MishaJexia) [reply docs/get-started/README.md:281]: fixed.

File: docs/ums/README.md (hunk 14 above)
Comment (MishaJexia) [reply docs/ums/README.md:64]: fixed

File: docs/ums/README.md (hunk 15 above)
Comment (MishaJexia) [reply docs/ums/README.md:199]: next version

File: docs/ums/README.md (hunk 16 above)
Comment (MishaJexia) [reply docs/ums/README.md:238]: next version

File: docs/ums/README.md (hunk 17 above)
Comment (MishaJexia) [reply docs/ums/README.md:276]: next ver

File: docs/ums/README.md (hunk 18 above)
Comment (MishaJexia) [reply docs/ums/README.md:243]: fixed

File: docs/ums/README.md (hunk 19 above)
Comment (MishaJexia) [reply docs/ums/README.md:327]: next release.

File: docs/ums/README.md (hunk 20 above)
Comment (MishaJexia) [reply docs/ums/README.md:341]: fixed

File: docs/ums/README.md (hunk 21 above)
Comment (MishaJexia) [reply docs/ums/README.md:402]: next release
